Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+ENV PYTHONUNBUFFERED 1
+RUN pip install poetry==1.0.0
+RUN mkdir /code
+ADD poetry.lock /code/
```

Comment:
ADD is older docker command more or less behaving the same as COPY, and it's left for in docker for backwards compatibility reasons. It's advised to prefer COPY over ADD in new projects phoenixnap.com/kb/docker-add-vs-copy